meinberlin/assets/scss/components/_button.scss

```diff
@@ -51,6 +51,8 @@
 }
 
 .btn--light {
+    font-weight: inherit;
```

Comment:
@fuzzylogic2000 is this really bad? we just seem to overwrite the bold font weight or always use btn light with small that I didn't want to make a new class name for it

Reply:
I think it's fine!